src/Client.php

```diff
@@ -183,24 +183,21 @@ public function getScheduleForDay(array $channelIds, Carbon $date)
             foreach ($data->items as $item) {
 
                 $scheduleItem = new ScheduleItem();
-
-                $scheduleItem->title = $item->title;
-                $scheduleItem->summary = $item->asset->summary->short;
-                $scheduleItem->episodeTitle = $item->asset->title;
-                $scheduleItem->dateTime = new Carbon($item->dateTime);
-
-                $scheduleItem->genres = collect();
-
-                foreach ($item->asset->tag as $tag) {
-                    if (str_contains($tag->id, 'genre:')) {
-                        $genreParts = explode(':', $tag->id);
-                        $scheduleItem->genres->push($genreParts[1]);
+                if (isset($item->title, $item->asset->summary, $item->asset->title, $item->dateTime, $item->asset->tag, $item->channel->id)) {
```

Comment:
You might wish to add `$item->asset` before `$item->asset->summary`, and then `$item->asset->summary->short`.